### PR TITLE
Upstream bug hunting vol. 2

### PR DESCRIPTION
**Pull request by Naxodile**
Ports fixes I made on Urist that never made it upstream
- Optables requires power to function
- Invisible soil?
- Applying pressure already affected internal bleeding, but it can now be done without needing bleeding
- Surgical wounds would set autoheal, but never reset it
- Windows could resist airflow regardless of construction state
- Attempting to talk when you can't tells you such
- Surgically healing organs no longer causes scarring (without any healing) of necrotic/decaying organs
- More grammar

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 1]
```diff
@@ -252,7 +252,7 @@
 */
 /mob/living/carbon/human/proc/apply_pressure(mob/living/user, var/target_zone)
 	var/obj/item/organ/external/organ = get_organ(target_zone)
-	if(!organ || !(organ.status & ORGAN_BLEEDING) || BP_IS_PROSTHETIC(organ))
+	if(!organ || !(organ.status & ORGAN_BLEEDING || organ.status & ORGAN_ARTERY_CUT) || BP_IS_PROSTHETIC(organ))
```
Comment (MistakeNot4892) [inline code/modules/mob/living/carbon/human/human_attackhand.dm]: These should be in separate sets of parens due to precedence issues with the `&` operator ie. ` !((organ.status & ORGAN_BLEEDING) || (organ.status & ORGAN_ARTERY_CUT))`

File: code/modules/mob/living/carbon/human/say.dm [hunk 2]
```diff
@@ -147,6 +147,7 @@
 
 /mob/living/carbon/human/handle_speech_problems(var/list/message_data)
 	if(HAS_STATUS(src, STAT_SILENCE) || (sdisabilities & MUTED))
+		to_chat(src, SPAN_DANGER("You are unable to speak!"))
```
Comment (MistakeNot4892) [inline code/modules/mob/living/carbon/human/say.dm]: `SPAN_WARNING` would be more appropriate here.

File: code/modules/organs/external/head.dm [hunk 3]
```diff
@@ -12,7 +12,7 @@
 	joint = "jaw"
 	amputation_point = "neck"
 	encased = "skull"
-	artery_name = "cartoid artery"
+	artery_name = "carotid artery"
```
Comment (MistakeNot4892) [inline code/modules/organs/external/head.dm:15]: wtf I'm having a Berenstain Bears moment. I would have sworn on any god you liked that it was cartoid.

File: code/modules/surgery/organs_internal.dm [hunk 4]
```diff
@@ -30,6 +30,9 @@
 	if(affected)
 		for(var/obj/item/organ/internal/I in affected.internal_organs)
 			if(I.damage > 0)
+				if(I.status & ORGAN_DEAD)
+					to_chat(user, SPAN_WARNING("\The [I.name] is [I.can_recover() ? "decaying" : "necrotic"] and cannot be treated with \the [tool] alone."))
```
Comment (MistakeNot4892) [inline code/modules/surgery/organs_internal.dm]: `\The [I]`

File: code/modules/mob/living/carbon/human/human_attackhand.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/mob/living/carbon/human/human_attackhand.dm]: Looks like you could also collapse the flag check to `!(organ.status & (ORGAN_BLEEDING|ORGAN_ARTERY_CUT))`

File: code/modules/surgery/organs_internal.dm [hunk 5]
```diff
@@ -58,13 +61,12 @@
 	var/obj/item/organ/external/affected = target.get_organ(target_zone)
 	for(var/obj/item/organ/internal/I in affected.internal_organs)
 		if(I && I.damage > 0 && !BP_IS_PROSTHETIC(I) && (I.surface_accessible || affected.how_open() >= (affected.encased ? SURGERY_ENCASED : SURGERY_RETRACTED)))
-			if(I.status & ORGAN_DEAD && I.can_recover())
-				user.visible_message("<span class='notice'>\The [user] treats damage to [target]'s [I.name] with [tool_name], though it needs to be recovered further.</span>", \
-				"<span class='notice'>You treat damage to [target]'s [I.name] with [tool_name], though it needs to be recovered further.</span>" )
+			if(I.status & ORGAN_DEAD)
+				to_chat(user, SPAN_NOTICE("You were unable to treat \the [I.name] due to its necrotic state."))
```
Comment (MistakeNot4892) [inline code/modules/surgery/organs_internal.dm]: `\the [I]`

Comment (Naxodile) [reply code/modules/surgery/organs_internal.dm]: [I]'ve gone ahead and removed the rest as well

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**